hbw/app/javascript/packs/hbw/components/shared/context/store.jsx

```diff
@@ -62,6 +63,7 @@ const withStoreContext = (WrappedComponent) => {
           }));
         } else {
           this.removeTaskFromList(taskId);
+          this.closeTaskOverview(taskId);
```

Comment:
Not here. If you complete/remove task you should close overview as well. Do it inside `removeTaskFromList`.